api-tests/relationships/nested-mutations/create-singular.test.js

```diff
@@ -42,21 +42,65 @@ function setupKeystone(adapterName) {
         },
       });
 
-      keystone.createList('GroupNoCreate', {
+      keystone.createList('GroupNoReadHard', {
+        fields: { name: { type: Text } },
+        access: { read: false },
+      });
+
+      keystone.createList('EventToGroupNoReadHard', {
         fields: {
-          name: { type: Text },
-        },
-        access: {
-          create: () => false,
+          title: { type: Text },
+          group: { type: Relationship, ref: 'GroupNoReadHard' },
         },
       });
 
+      keystone.createList('GroupNoCreate', {
+        fields: { name: { type: Text } },
+        access: { create: () => false },
```

Comment:
```suggestion
        fields: {
          name: { type: Text },
        },
        access: {
          create: () => false,
        },
```

Reply:
Cleaning up the diff to make it easier to asses exactly what changes are happening 👍 (FWIW I like the shorter version of the code, but it's making this diff really hard to read. Would definitely accept a separate clean-up PR to tighten everything up 👍 )